contracts/TeamVesting.sol

```diff
@@ -1,4 +1,5 @@
-pragma solidity 0.8.4;
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
```

Comment:
```suggestion
pragma solidity 0.8.4;
```
using the fixed solidity version was required by audit team, lets use 0.8.4 and keep all the same